Tests/UnitTests/Plugins/EDM4hep/ConvertTrackEDM4hepTest.cpp

```diff
@@ -365,7 +414,7 @@ BOOST_AUTO_TEST_CASE(RoundTripTests) {
                            readTs.smoothedCovariance(), 1e-5, 1e-6);
       Vector3 newCenter = readTs.referenceSurface().center(
           gctx);  // new center is a perigee, but should be on the other
-                  // surface
+      // surface
```

Comment:
I think the previous formatting was correct?

Reply:
I think that's ok like this as well.